config/assets/default.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
         'public/lib/angular-ui-router/release/angular-ui-router.js',
         'public/lib/angular-ui-utils/ui-utils.js',
         'public/lib/angular-bootstrap/ui-bootstrap-tpls.js',
```

Comment:
yeah it was working perfect without public/lib/angular-bootstrap/ui-bootstrap.js' also it is written in its own documentations that we need to decide  (ui-bootstrap.js  and some of templates ) or ui-bootstrap-tpls.js